src/BlockTypesController.php

```diff
@@ -191,7 +191,7 @@ protected function get_block_types() {
 		/**
 		 * This disables specific blocks in Widget Areas by not registering them.
 		 */
-		if ( in_array( $pagenow, [ 'widgets.php', 'themes.php', 'customize.php' ], true ) ) {
+		if ( in_array( $pagenow, [ 'widgets.php', 'themes.php', 'customize.php' ], true ) && ! empty( 'gutenberg-edit-site' !== $_GET['page'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
Just to clarify, is this `empty()` to account for the case that `$_GET['page']` is not defined? In that case, I think you should check first that `$_GET['page']` is defined and then do the comparison. Something like this:
```PHP
&& ( empty( $_GET['page'] ) || 'gutenberg-edit-site' !== $_GET['page'] )
```